src/server/handlers/github.js

```diff
@@ -46,12 +47,17 @@ const RESPONSE_CREATED = {
 };
 
 export const listRepositories = (req, res) => {
-  const uri = '/user/repos?affiliation=owner';
+  let params = ['affiliation=owner'];
 
   if (!req.session || !req.session.token) {
     return res.status(401).send(RESPONSE_AUTHENTICATION_FAILED);
   }
 
+  if (req.params.page) {
+    params.push('page=' + req.params.page);
+  }
+
+  const uri = REPOSITORY_ENDPOINT + '?' + params.join('&');
```

Comment:
Consider using `qs.stringify` instead.  Doesn't matter much here, but might save us from quoting errors in future.